tests/test_on_data/test_behavior_interfaces.py

```diff
@@ -333,17 +334,36 @@ class TestDeepLabCutInterface(DeepLabCutInterfaceMixin, unittest.TestCase):
 
     save_directory = OUTPUT_PATH
 
-    def check_run_conversion(self, nwbfile_path: str):
```

Comment:
This particular case showing how a general name like `check_run_conversion` can be misleading - this particular one was checking entirely different things from the parent, didn't call the parent, and could have been missed otherwise here as not actually checking a certain behavior

This can be a real problem with the amount of misdirection in the current mixin setup, but given the non-concurrent I/O for various interfaces I also cannot think of a better way